imdown-mobile/.gitignore

```diff
@@ -12,3 +12,70 @@ web-report/
 
 # macOS
 .DS_Store
+
+# The following contents were automatically generated by expo-cli during eject
```

Comment:
wow expo really wants whats best for us

Reply:
its like a kind parent but eventually we have to move out